src/Psalm/Internal/Analyzer/StatementsAnalyzer.php

```diff
@@ -1464,4 +1465,36 @@ public function getFunctionAnalyzers()
     {
         return $this->function_analyzers;
     }
+
+    /**
+     * @param  PhpParser\Node\Expr $first_arg_value
+     *
+     * @return null|string
+     */
+    public static function getConstName($first_arg_value)
+    {
+        $const_name = null;
+
+        if ($first_arg_value instanceof PhpParser\Node\Scalar\String_) {
+            $const_name = $first_arg_value->value;
+        } elseif (isset($first_arg_value->inferredType)) {
+            $possible_value_types = $first_arg_value->inferredType->getTypes();
+            if (count($possible_value_types) === 1) {
```

Comment:
There are convenience functions added by @TysonAndre  - `Psalm\Type\Union::hasSingleStringLiteral` && `Psalm\Type\Union::getSingleStringLiteral` that'll do this job for you

Reply:
Good to know. Now fixed.